pkg/server/auth/server.go

```diff
@@ -323,6 +323,10 @@ func (s *AuthServer) UserInfo() http.HandlerFunc {
 			return
 		}
 
+		if s.provider == nil {
+			return
+		}
```

Comment:
This fix + a whole load of auth tests is coming in https://github.com/weaveworks/weave-gitops/pull/1699, so you can drop this bit